lib/watch/tsc-watcher.js

```diff
@@ -56,9 +56,12 @@ function watchMain (args, opts) {
     }
     origPostProgramCreate(program)
 
+    // We need to set the NODE_PATH variable in case node_modules are in a child directory
+    const env = Object.assign({}, process.env, { NODE_PATH: resolve(process.cwd(), 'node_modules') })
```

Comment:
This scares me quite a lot. Why is it needed?

Reply:
It scares me too, but the problem is that we can have a parent directory with `node_modules` and a child directory having `node_modules` too.
If the `outDir` is called lib (for example) and it is built in the parent dir we have:
- parent
  -- child
    --- node_modules
    --- src
  -- node_modules
  -- lib 
    --- index.js
If in index.js we require something which is in child/node_modules it is not found.

I believe a more soft approach might be to append the directory to the `NODE_PATH` instead of replacing it. What do you think?

Reply:
I would not touch `NODE_PATH` in any way.